spec-dtslint/operators/min-spec.ts

```diff
@@ -0,0 +1,17 @@
+import { of } from 'rxjs';
+import { max } from 'rxjs/operators';
+
+it('should infer correctly', () => {
+    const a = of(1, 2, 3).pipe(max()); // $ExpectType Observable<number>
+    const b = of('abc', 'bcd', 'def').pipe(max()); // $ExpectType Observable<string>
+});
+
+it(' should except empty comparer', () => {
+    const a = of(1, 2, 3).pipe(max()); // $ExpectType Observable<number>
+});
+
+it('should infer on comparer types', () => {
```

Comment:
This isn't so much inferring as it is enforcing, so I'd have used `'should enforce comparer types'`.